docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -112,9 +115,9 @@ The examples below are using the desktop and mobile variants of the pickers, but
   ```
 
   - Default `views` prop: `['hours', 'minutes']` (plus a `meridiem` view if the locale is in 12-hours format)
-  - Default `closeOnSelect` prop: `true`
+  - Default `closeOnSelect` prop: `false`
 
-  **Behavior:** The picker will close when selecting the minutes or meridiem (if a 12-hour clock is used).
+  **Behavior:** The picker won't close when selecting the minutes or meridiem (if a 12-hour clock is used).
```

Comment:
Nitpick

```suggestion
  **Behavior:** The picker will not close when selecting the minutes or meridiem (if a 12-hour clock is used).
```

It seems that we are using both but "will not" a lot more than "won't"

Reply:
Replaced all cases of `won't` in this file.